### PR TITLE
Add support for location scenario reference

### PR DESCRIPTION
**Pull request by freak4pc**
This PR parses a launch action's location scenario reference - the portion which defines the simulated GPS location of a runnable scheme

I did my best to assume what are the right tests here, but I'm not a ruby developer so if I missed anything please do let me know :) 

Thanks!

### CODE REVIEW

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! <br />
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* Add support for location scenario reference  
  [freak4pc](https://github.com/freak4pc)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by freak4pc** [comment]
Sorry for the ping but any chance to look at this? We want to add some testing to a different OSS project that needs this reference exposed. @dnkoutso 🙏

File: lib/xcodeproj/scheme/location_scenario_reference.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
+module Xcodeproj
+  class XCScheme
+    # This class wraps the LocationScenarioReference node of a .xcscheme XML file
+    #
+    # A LocationScenarioReference is a reference to a simulated GPS locatio associated
```
Comment (dnkoutso) [inline lib/xcodeproj/scheme/location_scenario_reference.rb:5]: typo nit: location

**Review by dnkoutso — APPROVED** [review]

File: lib/xcodeproj/scheme/location_scenario_reference.rb (hunk 1 above)
Comment (freak4pc) [reply lib/xcodeproj/scheme/location_scenario_reference.rb]: Fixed, thank you @dnkoutso !

**Comment by freak4pc** [comment]
ready for you when you can :) @dnkoutso

**Comment by freak4pc** [comment]
Any chance to get this pulled in to unblock a different PR waiting for it? @amorde @dnkoutso

**Comment by igor-makarov** [comment]
LGTM!
@segiddins @dnkoutso?

**Review by dnkoutso — APPROVED** [review]

**Comment by freak4pc** [comment]
The approval isn't the issue (you already did that) I just don't have merging permissions unfortunately :) 

@dnkoutso

**Comment by freak4pc** [comment]
Thank you! Any plans on cutting a release soon? Seems like the last one was 6 months ago, based on GitHub at least?

**Comment by Zanchee** [comment]
Also hoping to get a release soon as well! I would love to start using a few things that have been added since the last release.

**Comment by dnkoutso** [comment]
released with 1.20.0

**Pull request merged**